tailwind.config.js

```diff
@@ -61,133 +61,6 @@ module.exports = {
                 "safe-left": "env(safe-area-inset-left)",
                 "safe-right": "env(safe-area-inset-right)",
             },
-            typography: {
```

Comment:
@tailwindcss/typography is not removed. You may need to remove it fromt he bottom of the file